components/scroller/scroller_default.story.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div>
+    <h3>Scroller</h3>
+
+    <button @click="scrollToItem(100)">
+      Go to index 100
+    </button>
+    <button @click="scrollToItem(0)">
+      Go to start
+    </button>
+    <button @click="scrollToItem(items.length - 1)">
+      Go to end
+    </button>
+
+    <dt-scroller
+      ref="scroller"
+      class="scroller"
+      :items="items"
+      :item-size="$attrs.itemSize"
+      :scroller-height="$attrs.scrollerHeight"
+      :scroller-width="$attrs.scrollerWidth"
+      :min-item-size="$attrs.minItemSize"
+      :list-tag="$attrs.listTag"
+      :item-tag="$attrs.itemTag"
+      :direction="$attrs.direction"
+      key-field="id"
+    >
+      <template #default="{ active }">
+        <div class="user">
+          active: {{ active }}
+        </div>
+      </template>
```

Comment:
hmm can we try an example using dt-list-item? I expect this is what would be used within the scroller the majority of the time.

Reply:
Mhmm I dont see necessary to wrap a bunch of code if there is no special usage.
dt-list-item is used on dynamic-size to handle size computations of the element.

There is not necessary to use it if you just want to use a simple list.

Reply:
what about keyboard navigation?

Reply:
We dont have any keyboard navigation here... either in the library...
Im guessing because there is no way you want to navigate between 1000 items with keyboard...

But... it is possible for the developer to build a manual keyboard navigation.
For example: 

```
switch (e.key) {
      focusedOptionIndex.value++;
      break;
    case 'ArrowUp':
      focusedOptionIndex.value--;
      break;
    case 'Home':
      focusedOptionIndex.value = 0;
      break;
    case 'End':
      focusedOptionIndex.value = props.options.length - 1;
      break;
    case 'Escape':
      opened.value = false;
      focusedOptionIndex.value = null;
      break;
    case 'Enter':
      onSelect();
      break;
  }
```

Reply:
I dont exactly why you ask for keyboard navigation on dtItemList discussion, do you have sth in mind?

Reply:
We have an existing mixin that works with our dt-list-items. Anyways it's not really part of the requirements for this MVP so I don't think we need to worry about it for now, though I know they will want it in the future.